bridge/svix-bridge/src/main.rs

```diff
@@ -121,18 +118,14 @@ fn setup_tracing(cfg: &Config) {
     };
 }
 
-pub fn setup_metrics(cfg: &Config) -> Option<BasicController> {
+pub fn setup_metrics(cfg: &Config) -> Option<MeterProvider> {
     cfg.opentelemetry.as_ref().map(|otel_cfg| {
         let exporter = opentelemetry_otlp::new_exporter()
             .tonic()
             .with_endpoint(&otel_cfg.address);
 
         opentelemetry_otlp::new_pipeline()
-            .metrics(
-                selectors::simple::inexpensive(),
-                delta_temporality_selector(),
-                Tokio,
-            )
+            .metrics(Tokio)
```

Comment:
It seems like those selectors have no clear equivalents in the latest version of opentelemetry_otlp / opentelemetry_sdk. However, [the PR that introduced these metrics](https://github.com/svix/svix-webhooks/pull/1051) provided no explanation for why those selectors were chosen. Thus I *guess* the defaults should be fine?

Reply:
I think these selectors are now applied using [with_aggregation_selector](https://docs.rs/opentelemetry-otlp/latest/opentelemetry_otlp/struct.OtlpMetricPipeline.html#method.with_aggregation_selector) and [with_temporality_selector](https://docs.rs/opentelemetry-otlp/latest/opentelemetry_otlp/struct.OtlpMetricPipeline.html#method.with_temporality_selector)

Reply:
Yeah I found that, but I dind't find an "inexpensive" aggregation selector or a "delta" temporality selector.

Reply:
This doesn't answer your question, but it reminded me of something random I've seen in Datadog docs (we use datadog for reporting metrics): https://docs.datadoghq.com/opentelemetry/guide/otlp_delta_temporality/

Reply:
Hmm, well [`DefaultTemporalitySelector`s documentation](https://docs.rs/opentelemetry_sdk/0.21.1/opentelemetry_sdk/metrics/reader/struct.DefaultTemporalitySelector.html) says that

> [Temporality::Cumulative](https://docs.rs/opentelemetry_sdk/0.21.1/opentelemetry_sdk/metrics/data/enum.Temporality.html#variant.Cumulative) will be used for all instrument kinds if this [TemporalitySelector](https://docs.rs/opentelemetry_sdk/0.21.1/opentelemetry_sdk/metrics/reader/trait.TemporalitySelector.html) is used.

And of the other two option, `ManualReader` seems to wrap other selectors and not have any relevant behavior of its own, so that seems like a hint that [`PeriodicReader`](https://docs.rs/opentelemetry_sdk/0.21.1/opentelemetry_sdk/metrics/struct.PeriodicReader.html) might be more appropriate here.

Reply:
Opened https://github.com/open-telemetry/opentelemetry-rust/discussions/1511

Reply:
Restored delta temporality selector as recommended upstream. I think this is now ready to merge.